myuw/views/api/instructor_schedule.py

```diff
@@ -53,6 +54,20 @@ def make_http_resp(self, timer, term, request, summer_term=None):
         """
         schedule = get_instructor_schedule_by_term(term)
         resp_data = load_schedule(request, schedule)
+        threads = []
+
+        for section in resp_data['sections']:
+            _set_current_or_future(term, request, section)
+            section_label = section['section_label']
+            t = Thread(target=coda.get_course_card_details,
+                       args=(section_label.replace("_", "-"),
```

Comment:
If this change applies to all the section label, please make the change in safe_label()